weblab/experiments/tests/test_models.py

```diff
@@ -20,11 +20,15 @@ class TestExperiment:
     def test_name(self, helpers):
         model = recipes.model.make(name='my model')
         protocol = recipes.protocol.make(name='my protocol')
+        model_version = helpers.add_version(model, tag_name='v1')
+        protocol_version = helpers.add_version(protocol, tag_name='v2')
+        populate_entity_cache(model)
```

Comment:
`add_version` should populate the cache, so you shouldn't need these calls.

Reply:
done thanks